cspyce/cspyce1.py

```diff
@@ -1243,6 +1241,10 @@ def _vectorize_arglist(arglist, k):
 #     globals()[sename] = efunc
 #     globals()[vfname] = vfunc
 
+
+
+del func, efunc, vfunc, vefunc
```

Comment:
This is an odd thing to do. Are you just trying to not make them visible to the outside world? In which case putting an underscore in front would work better.